### DISCUSSION TITLE
Astro sveltia starter & Astro integration

### DISCUSSION THREAD

**Discussion by zanhk** [sveltia/sveltia-cms#75] [Show and tell]
If this could help anybody lately I ve migrated my [astro starter template](https://github.com/zankhq/astro-starter) to Sveltia CMS, As soon as I will have some free time I ll work on an [astro sveltia intragration](https://github.com/majesticostudio/astro-sveltia-cms) with the aim to somplify the setup of new astro sveltia projects togheter with a step by step guide on how to get started on cloudflare and netlify.

**Comment by kyoshino** [comment]
Cool, I’ll add your starter kit to README 😀

**Comment by zanhk** [comment]
Moved https://github.com/zankhq/astros to sveltia cms as well!

**Reply by kyoshino** [reply]
Updated README to list all your projects!
